### PR TITLE
ReadMe

### PR DESCRIPTION
**Pull request by Micjohn01**

### CODE REVIEW

**Review by ro61zzy — APPROVED** [review]
looks good

File: README.md
Comment (michojekunle) [inline README.md:1]: - Looks great, well-done bro, will merge these, however for the setting up part, the git URL needs to be updated to use this GitHub repository URL 
- also, some things will still need to be reviewed in this documentation as we proceed
- and lastly, we need to add the team members and their roles in this documentation

**Review by michojekunle — APPROVED** [review]
- Looks great, well-done bro, will merge these, however for the setting up part, the git URL needs to be updated to use this GitHub repository URL
- also, some things will still need to be reviewed in this documentation as we proceed
- and lastly, we need to add the team members and their roles in this documentation

File: README.md
Comment (Micjohn01) [reply README.md:1]: I will get it fix now and raise a PR.

**Pull request merged**